### PR TITLE
feat: Only include basis data from succeeded calculations in settlement reports

### PR DESCRIPTION
**Pull request by JesperGomes**
# Description

Added comments to two views.
A confluence page was create to describe commen view columns: 
https://energinet.atlassian.net/wiki/spaces/D3/pages/841744394/Column+Descriptions

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [ ] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql [hunk 1]
```diff
@@ -0,0 +1,29 @@
+DROP VIEW IF EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_periods_v1
+GO
+
+CREATE VIEW IF NOT EXISTS {SETTLEMENT_REPORT_DATABASE_NAME}.metering_point_periods_v1
+     (calculation_id,
+    calculation_type COMMENT '\'BalanceFixing\' | \'Aggregation\' | WholesaleFixing | FirstCorrectionSettlement | SecondCorrectionSettlement | ThirdCorrectionSettlement',
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql:6]: Why only \' on the first two?

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    metering_point_id,
+    from_date,
+    to_date COMMENT '<value> | NULL',
+    grid_area_code,
+    from_grid_area_code COMMENT '<value> | NULL',
+    to_grid_area_code COMMENT '<value> | NULL',
+    metering_point_type COMMENT 'production | consumption | exchange | ve_production | ve_consumption | ve_exchange | ve_total_consumption | ve_non_profiled_consumption | ve_flex_consumption',
```
Comment (JesperGomes) [inline source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql:13]: these does not match this: 
https://github.com/Energinet-DataHub/opengeh-wholesale/blob/main/source/databricks/calculation_engine/package/codelists/metering_point_type.py

Why is that?

Comment (JesperGomes) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql:13]: this comment applies to the other script as well

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql (hunk 1 above)
Comment (energinet-ajw) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql:6]: I was testing. I was difficult to find the correct syntax. A later check-in fixes this.

File: source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql (hunk 2 above)
Comment (energinet-ajw) [reply source/databricks/calculation_engine/package/datamigration/migration_scripts/202405131200_Modify_settlement_report_metering_point_periods_v1_view.sql:13]: Co-pilot helped me... :(

**Review by energinet-ajw — APPROVED** [review]

**Review by djorgensendk — APPROVED** [review]

**Pull request merged**